### ISSUE TITLE
Scene clipping works only for first selected viewpoint

### ISSUE THREAD

**Issue by mst-lyubimov** [firemodels/smv#1600]
[ini.txt](https://github.com/firemodels/smv/files/11682108/ini.txt)
[smv.txt](https://github.com/firemodels/smv/files/11682110/smv.txt)

Attached files "smv" and "ini" (I had to change the extension to "txt", as GitHub does not support files with the extension "smv" and "ini")
There are 2 viewpoints defined "ini" file with different parameters of scene clipping. For point Level_F0_0 on Z axis the scene is clipped from 0 to 3, and for point Level_F1_3000 from 3 to 6.
I noticed that in smokeview 6.8.0 scene along Z axis will be clipped as specified for the first selected viewpoint.
In other words, if after starting smokeview first select Level_F0_0 and then Level_F1_3000, the scene will be clipped from 0 to 3, though it should be from 3 to 6.
Similarly, if you select Level_F1_3000 and then Level_F0_0 after starting smokeview, the scene will be clipped from 3 to 6, although it should be from 0 to 3.

**Comment by gforney** [comment]
Ill take a look when I get back

**Comment by mst-lyubimov** [comment]
In 6.7.21 the problem already present.
In 6.7.19 there is no problem.

**Comment by obscureed** [comment]
I think this is related to https://github.com/firemodels/smv/issues/1455, so this could be a chance to close two issues for the price of one. I agree with @mst-lyubimov that not all the changes between 6.7.18 and 6.7.21 were helpful. (I think there was at least one helpful change in this area around then, to do with whether the clipping dialog box remembered and displayed the current or most recent values. I can't find a github issue relating to this.) One good behavior before then was that clipping status was saved with each view, *including* absence of clipping. I think the problem in that earlier issue was about absence of clipping not being reimposed. The current issue seems to involve different behavior.

In the previous issue, I tried to work out what most users would expect if a saved view used clipping status X and was set to Apply at Startup, but the .ini settings were saved with clipping status Y. Some users would expect X, and some might expect Y. I've now gravitated towards X: the view includes clipping status, and that view applies at startup. That would suit me.

**Comment by gforney** [comment]
try test smokeview's posted here.  I've addressed similar problems before so hopefully the game of whackamole is converging on a solution.  let me know if some other problem pops up
https://github.com/firemodels/test_bundles/releases

**Comment by gforney** [comment]
did this fix the problem you reported?

**Comment by mst-lyubimov** [comment]
Yes, in build SMV-6.8.0-184 broblem is fixed. Thanks.
